src/main/java/org/jabref/gui/DragAndDropDataFormats.java

```diff
@@ -11,6 +11,7 @@
  */
 public class DragAndDropDataFormats {
 
+    public static final DataFormat FIELD = new DataFormat("dnd/org.jabref.model.entry.field.Field");
```

Comment:
For this JabRef-intern drag&drops, the https://github.com/JabRef/jabref/blob/master/src/main/java/org/jabref/gui/util/CustomLocalDragboard.java is easier to use.